### ISSUE TITLE
Deprecation warnings in Sass 3.3.0.rc

### ISSUE THREAD

**Issue by lmartins** [at-import/Singularity#137]
You are probable well aware of this by now but here it goes anyway.

When compiling using the new rc versions of Sass and Compass we get a lot of deprecation warnings like this:

```
DEPRECATION WARNING on line 26 of /Users/luismartins/.rvm/gems/ruby-2.0.0-p247/gems/singularitygs-1.1.2/stylesheets/singularitygs/helpers/_layout.scss:
Assigning to global variable "$gutters" by default is deprecated.
In future versions of Sass, this will create a new local variable.
If you want to assign to the global variable, use "$gutters: $layout-private-gutter-holder !global" instead.
```

**Comment by monsdroid** [comment]
Hi,

just wantet to add that with compass 0.13alpha and sass 3.3.0alpha your framework wont work at all.
as soon as i '@include grid-span()' the calculation fails.

With sass 3.2 and compass 0.12 the same code works.
example:

```
//////////////////////////////
// Imports
//////////////////////////////
@import "singularitygs";

// Breakpoints
//mObile tAblet dEsktop vErywide
//only guessed
$bp-m:  0em; $col-m:  4; $gut-m: 1/10;
$bp-t:  30em; $col-t: 12; $gut-t: 3/10;
$bp-d:  60em; $col-d: 16; $gut-d: 5/10;
$bp-v:  90em; $col-v: 24; $gut-v: 9/10;

// Mobile First 
$grids: $col-m;
$gutters: $gut-m;

/* Dummy Grids */
$grids:   add-grid( $col-t at $bp-t );
$gutters: add-gutter( $gut-t at $bp-t );

$grids:   add-grid( $col-d at $bp-d );
$gutters: add-gutter( $gut-d at $bp-d );

$grids:   add-grid( $col-v at $bp-v );
$gutters: add-gutter( $gut-v at $bp-v );

body,html {
  min-height: 100%;
  width:100%;
  margin: 0;
  padding: 0;

  @include background-grid( $col-m, $gut-m );

  @include breakpoint($bp-t){
    @include background-grid( $col-t, $gut-t );
  }
  @include breakpoint($bp-d){
    @include background-grid( $col-d, $gut-d );
  }
  @include breakpoint($bp-v){
    @include background-grid( $col-v, $gut-v );
  }
}

img {
  @include grid-span(4,1);

  @include breakpoint($bp-t){
    @include grid-span(8,2);
  }

  @include breakpoint($bp-d){
    @include grid-span(10,3);
  }
}
```

Are you Planning on Supporting the new Sass and Compass Version?
If yes, can you guess when?

**Comment by Snugug** [comment]
Yah, we know. We're a little delayed with the update due to work and conference schedules. Soon though, I promise.

**Comment by chrisspiegl** [comment]
Is there a way to tell compass to use the old version of sass? Till we wait for the update.

**Comment by Snugug** [comment]
Bundler

**Comment by chrisspiegl** [comment]
@Snugug and which version of sass should we be using? I can not find it anywhere (sorry).

**Comment by Snugug** [comment]
`gem "sass", "~>3.2.0"`

**Comment by chrisspiegl** [comment]
Thanks, @Snugug.

Hope this amazing grid system (the first I like using) will get the update to SASS 3.3 soon.

**Comment by chrisspiegl** [comment]
Just wondered about another thing: Is there a roadmap to support 3.3?

**Comment by Snugug** [comment]
[v1.2.0](https://github.com/Team-Sass/Singularity/issues?milestone=4&page=1&state=open) will be Sass 3.3 compatible.

**Comment by Snugug** [comment]
Resolved in #157  (which will become v1.2.0 of Singularity)
